### DISCUSSION TITLE
UNASSIGNED Shards

### DISCUSSION THREAD

**Discussion by cyberkimh** [Security-Onion-Solutions/securityonion#8156] [Unsupported Versions]
Hello, I was wondering if anyone could help with deleting UNASSIGNED shards please?
We've just rebooted all the nodes and a couple of shards need to be deleted as they're unassigned. We usually use:

`curl -k -XGET https://localhost:9200/_cat/shards | grep UNASSIGNED`
 to find which are still unassigned, and then:

`curl -k -XGET https://localhost:9200/_cat/shards | grep UNASSIGNED | awk {'print $1'} | xargs -i curl -k -XDELETE "https://localhost:9200/{}"`
to delete them and then run:

`so-elastic-restart`

But there is still one left unassigned and I can't delete it for some reason.

![image](https://user-images.githubusercontent.com/61547463/174599703-a16bf7b3-eb1c-46cf-8c7b-6dad396753fa.png)

I'd appreciate any help, thanks.

**Comment by weslambert** [comment]
You might consider just deleting the whole index if you don't need it.
